haskell/src/SwiftNav/SBP/Acquisition.hs

```diff
@@ -0,0 +1,26 @@
+module SwiftNav.SBP.Acquisition where
+
+import Data.Int
+import Data.Text
+import Data.Word
+
+
```

Comment:
Are you defining an SBP typeclass or something that deals with the serialization/deserialization/packing, etc.?